auction_keeper/main.py

```diff
@@ -216,6 +223,14 @@ def check_cdps(self):
             rate = ilk.rate
             safe = current_urn.ink * ilk.spot >= current_urn.art * rate
             if not safe:
+                if self.vat.dai(self.our_address) == Rad(0):
```

Comment:
How much Dai does the account need for a bite? will `0.01` DAI versus `1` DAI work? Since it's an Ethereum method call I thought it was dependent on the amount of Ether in your account to make the method call i.e. just amount / cost dependent?

Could you expand on what's required on behalf of the user to call the bite method?

Reply:
0 Dai are required to bite, but to bid, they need Dai > 0.  In a prior code review, George made the point that the user shouldn't waste gas biting if there's no chance they can participate in the auction.